examples/asr/asr_adapters/train_asr_adapter.py

```diff
@@ -120,7 +121,7 @@ def main(cfg):
         raise ValueError("Cannot set `cfg.model.nemo_model` and `cfg.model.pretrained_model`. Select one only.")
```

Comment:
I'd write "Cannot set both `cfg.model.nemo_model` and `cfg.model.pretrained_model`. Set one only."